### PR TITLE
Re-name really long test file

### PR DESCRIPTION
**Pull request by orta**
When I cloned this repo in a Windows VM, I got  an issue that my file system can't create the file due to the length of its name. Given that the name is auto-generated, I gave it a  shorter name and left the full name in a comment. 

Screen Shot 2021-06-03 at 4 48 59 PM

### CODE REVIEW

File: src/testRunner/unittests/tsserver/projectReferences.ts [hunk 1]
```diff
@@ -676,8 +676,10 @@ instance.fly();`,
                 "fly"
             );
 
+
             verify(
-                "when using arrow function as object literal property is loaded through indirect assignment with original declaration local to project is treated as local",
+                // when using arrow function as object literal property is loaded through indirect assignment with original declaration local to project is treated as local
+                "when using arrow function as object literal property...",
```
Comment (sheetalkamat) [inline src/testRunner/unittests/tsserver/projectReferences.ts:682]: can you remove `...` in the end. 
Thanks

**Comment by weswigham** [comment]
`git config --system core.longpaths true` has been required to clone either this repo or the internal repo from time to time 😄

**Comment by orta** [comment]
Heh, I had renamed the file to be the same as another test which was confusing the baselines - fixed that 👍🏻

**Comment by morzel85** [comment]
Here's a list of files with dangerously long paths (above 200, raging from 223 to 202 chars):

- tests/baselines/reference/tscWatch/emitAndErrorUpdates/incremental/assumeChangesOnlyAffectDirectDependenciesAndD/updates-errors-when-file-transitively-exported-file-changes/when-there-are-no-circular-import-and-exports.js
- tests/baselines/reference/tscWatch/emitAndErrorUpdates/incremental/assumeChangesOnlyAffectDirectDependenciesAndD/updates-errors-when-file-transitively-exported-file-changes/when-there-are-circular-import-and-exports.js
- tests/baselines/reference/tscWatch/emitAndErrorUpdates/incremental/assumeChangesOnlyAffectDirectDependencies/updates-errors-when-file-transitively-exported-file-changes/when-there-are-no-circular-import-and-exports.js
- tests/baselines/reference/tsserver/projectReferences/solution-with-its-own-files-and-disables-looking-into-the-child-project-if-disableReferencedProjectLoad-is-set-in-first-indirect-project-but-not-in-another-one.js
- tests/baselines/reference/tscWatch/emitAndErrorUpdates/incremental/assumeChangesOnlyAffectDirectDependencies/updates-errors-when-file-transitively-exported-file-changes/when-there-are-circular-import-and-exports.js
- tests/baselines/reference/tscWatch/emitAndErrorUpdates/assumeChangesOnlyAffectDirectDependenciesAndD/updates-errors-when-file-transitively-exported-file-changes/when-there-are-no-circular-import-and-exports.js
- tests/baselines/reference/tscWatch/emitAndErrorUpdates/assumeChangesOnlyAffectDirectDependenciesAndD/updates-errors-when-file-transitively-exported-file-changes/when-there-are-circular-import-and-exports.js
- tests/baselines/reference/tscWatch/emitAndErrorUpdates/assumeChangesOnlyAffectDirectDependencies/updates-errors-when-file-transitively-exported-file-changes/when-there-are-no-circular-import-and-exports.js
- tests/baselines/reference/tscWatch/emitAndErrorUpdates/assumeChangesOnlyAffectDirectDependencies/updates-errors-when-file-transitively-exported-file-changes/when-there-are-circular-import-and-exports.js
- tests/baselines/reference/tsserver/projectReferences/monorepo-like-with-symlinks-when-packageJson-has-types-field-and-has-index.ts-with-scoped-package-and-solution-is-not-built-with-preserveSymlinks.js

**Comment by morzel85** [comment]
Long paths on Windows bit me a few times in my life hence I see them as "dangerous" ;) One would wish that in 2021 that would never cause any issue...  but for example: I've just checked and on my Win 10 Home Explorer shows a file I've created (by bash) under long path but i cannot rename it (changing registry setting _could_ help...). This is just a quick example showing that long path handling on Win is so-so.

**Pull request merged**